apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -68,10 +67,13 @@ const SetUpMessage = ({
   isRtl,
   headingText,
   descriptionText,
+  className,
   buttonText,
   buttonUrl,
+  buttonClass,
+  onClick,
 }) => (
-  <div style={styles.section}>
+  <div style={styles.section} className={className}>
```

Comment:
:( Makes we wish we could use enzyme style selectors in UI tests, i.e. `Foo.find('Button')`